contrib/intel/jenkins/summary.py

```diff
@@ -530,27 +530,37 @@ def __init__(self, logger, log_dir, prov, mpi, file_name, stage_name):
         super().__init__(logger, log_dir, prov, file_name, stage_name)
 
         self.mpi = mpi
-        if self.mpi == 'impi':
-            self.run = '/mpiexec'
-        else:
-            self.run = '/mpirun'
-
+        self.run = 'mpiexec'
+    def read_file(self):
+        previous = ""
+        with open(self.file_path,'r') as log_file:
+            for line in log_file:
+                line = line
+                super().check_features(previous, line.lower())
+                super().check_node(line.lower())
+                if "Env includes" in line:
+                    test_line = log_file.readline()
+                    result_line = log_file.readline()
+                    self.check_name(test_line)
+                    self.check_pass(result_line)
+                    self.check_fail(result_line)
+                previous = line
+    
     def check_name(self, line):
         if self.run in line:
             self.name = line.split()[len(line.split()) - 1].split('/')[1]
-            #assume pass
+    def check_pass(self, line):
+        if "no errors" in line:
             self.passes += 1
             self.passed_tests.append(self.name)
 
     def check_fail(self, line):
         # Fail cases take away assumed pass
-        if "exiting with" in line:
+        if "exited without no errors" in line:
```

Comment:
Is there a way to change the output to either "without errors" or "with errors", so we avoid the double negative and poor grammar that comes from "without no"?

Reply:
I agree about the poor  grammar. This output is however, generated by the runtests scripts which is provided by mpich to run the tests. We will have to manually/through CI code rewrite into this script file through if we want to change this.

Reply:
@shefty new logs say "ok" and "not ok' for pass/fail.

Reply:
@zachdworkin I'm not following what log you're referring to.  I'm used to searching out CI logs for 'failure'.  That's fine.  I was referencing the search check above.  I was assuming the change there was aligned with whatever output was being produced and if we could change that output.  If that output is coming from some other app not under our control, as Nikhil indicates, then I guess we have to live with the confusion.  Because I don't know if 'without no errors' is intended to mean that things passed or failed.

Reply:
@shefty the code shown is outdated. The new way of checking for failures specifically for mpich is using the "ok" "not ok" prints that come from the mpich dictated output. I was trying to document why I was resolving this conversation.